awsDockerrunTemplate.json

```diff
@@ -3,11 +3,11 @@
   "containerDefinitions": [
     {
       "name": "app",
-      "image": "REPOSITORY_URI:server_TAG_SUFFIX",
+      "image": "REPOSITORY_URI:serverTAG_SUFFIX",
```

Comment:
Could not get this working with the underscore before TAG_SUFFIX (even when trying to deploy master)

Reply:
weird... they're making it into the aws registry fine. This works to tho